src/MSALWrapper/AuthFlow/AuthFlowBase.cs

```diff
@@ -54,6 +54,7 @@ public async Task<AuthFlowResult> GetTokenAsync()
         /// The inner method required to be implemented by an AuthFlow.
         /// Performs token acquisition.
         /// </summary>
+        /// <param name="account">The <see cref="IAccount"/> to attempt to use.</param>

```

Comment:
Is this added by mistake? I do not see any param as `account`.

Reply:
Yes, removing in `[9/🧹]` :), good catch!